packages/main/src/RPA/Smartsheet.py

```diff
@@ -0,0 +1,1415 @@
+    def refresh_sheet(self, native: bool = False) -> Union[TableType, Sheet]:
+        """Refreshes the current sheet from the API and returns it
+        either as a Table or native data model depending on the
+        ``native`` argument.
+        """
+        self._require_current_sheet()
+        sheet = self.smart.Sheets.get_sheet(self.current_sheet.id)
+        self.current_sheet = sheet
+        if native:
+            return sheet
+        else:
+            return self.convert_sheet_to_table(sheet)
+
+    @keyword
+    def get_sheet_owner(
+        self, sheet_id: int = None, sheet_name: int = None
+    ) -> Tuple[str, int]:
+        """Returns the owner's username and ID for the current sheet."""
+        sheet_id = self._parse_sheet_id(sheet_id, sheet_name)
+        sheet = self.smart.Sheets.get_sheet(sheet_id, include=["ownerInfo"])
+        return sheet.owner, sheet.owner_id
+
+    @keyword
+    def list_sheet_filters(
+        self, sheet_id: int = None, sheet_name: int = None
+    ) -> List[SheetFilter]:
+        """Returns a list of available filters for the current sheet. You
+        can specify a different sheet via the ``sheet_id`` or
+        ``sheet_name`` parameters.
+
+        The returned list of filters can be used with the ``filter_id``
+        argument of the ``get_sheet`` keyword.
+
+        Example:
+
+        .. code-block:: robotframework
+
+            ${filters}=  List Sheet Filters
+            FOR  ${filter}  IN  @{filters}
+                ${filtered_sheet}=  Get Sheet
+                ...  sheet_name=My sheet
+                ...  filter_id=${filter.id}
+                Log  There are ${len(filtered_sheet)} rows in the filtered sheet
+
+        .. code-block:: python
+
+            ss = Smartsheet(access_token=access_token)
+            my_sheet_id = 123456789
+            filters = ss.list_sheet_filters()
+            for filter in filters:
+                filtered_sheet = ss.get_sheet(
+                    sheet_id=my_sheet_id,
+                    filter_id=filter.id,
+                    native=True,
+                )
+                print(
+                    f"There are {len(filtered_sheet.rows)} rows in the "
+                    f"filtered sheet"
+                )
+        """
+        self._require_current_sheet()
+        sheet_id = self._parse_sheet_id(sheet_id, sheet_name)
+        filter_result = self.smart.Sheets.list_filters(
+            sheet_id or self.current_sheet.id, include_all=True
+        )
+        return self._unpack_index_result(filter_result)
+
+    @keyword
+    def create_sheet(
+        self,
+        name: str,
+        columns: List[ColumnType] = None,
+        from_sheet_id: Union[int, str] = None,
+    ) -> Sheet:
+        """Creates a new sheet with the given name and columns, then sets
+        the current sheet to the new sheet and returns it as a native
+        Smartsheet object.
+
+        :param from_sheet_id: Sheet ID to use as a template for the new
```

Comment:
You forgot to document `name` and `columns` params.

Reply:
Interesting, again, copilot helped me write most of the docs... 🤔